draft-ietf-rtcweb-jsep.xml

```diff
@@ -4076,6 +4076,65 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             process, as a remote offer or answer, unmodified SDP coming
             from any other implementation of this specification.</t>
         </section>
+        <section title="Processing RTP packets"
+                 anchor="sec.rtp.demux">
+          <t>When an RTP packet is received by a transport (typically a
+          DtlsTransport), that packet MUST be routed to the correct
+          RtpReceiver. For each transport, the following steps MUST be
+          followed to prepare to route packets:</t>
+
+          <t>
+            <list>
+              <t>Construct a table mapping MID to RtpReceiver. For each
```

Comment:
Same here: transceiver.mid is defined in the W3C doc, but that's just a reflection of what's in the offer description (local or remote), so it seems like we ought to be able to point back to this document for those.